constants/events.go

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2022 LimeChain Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package constants
+
+import "github.com/gookit/event"
+
+const (
+	EventBridgeConfigUpdate          = "config.bridge.update"
+	BridgeConfigUpdateEventParamsKey = "params"
+	ClientsEventPriority             = event.Max
+	AssetServicePriority             = event.High
+	ServiceEventPriority             = event.AboveNormal
+	WatcherEventPriority             = event.Normal
+	HandlerEventPriority             = event.BelowNormal
```

Comment:
Are we using the `HandlerEventPriority` const somewhere?

Reply:
No, but I left it there to be easy for the person who will need to implement an event to know which priority it needs to be.